numpy/core/src/multiarray/compiled_base.c

```diff
@@ -112,20 +113,30 @@ arr_bincount(PyObject *NPY_UNUSED(self), PyObject *args, PyObject *kwds)
     }
     len = PyArray_SIZE(lst);
 
+    /*
+     * This if/else if can be removed by changing the argspec to O|On above,
+     * once we retire the deprecation
+     */
     if (mlength == Py_None) {
-        minlength = 0;
```

Comment:
OK, it is initialized to zero above. Might be clearer to have that here also as it makes explicit the current default value.